test/modules/packages/ZMQ/interop-py/wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+pyscript=$1
+
+if [[ -f pyzmq-venv/bin/activate ]] ; then
+    source pyzmq-venv/bin/activate
+    python $pyscript
+    deactivate
```

Comment:
I think the parent venv isn't affected because the `pyzmq-venv` is used only within a new Bash process when `wrapper.sh` is executed via `spawn()`.